index.js

```diff
@@ -116,7 +118,17 @@ function renderBundle(options, cb) {
         out += "module.exports = ";
       }
       out += sources[canonicalSourcePath];
-      out += "\n}, " + JSON.stringify(depMap[canonicalSourcePath]) + "],";
+      out += "\n}, {";
+      // for reproducibility, as JSON.stringify(depMap[canonicalSourcePath]) does not guarantee order - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify
+      var depMapKeys = Object.keys(depMap[canonicalSourcePath]);
+      depMapKeys.sort();
+      depMapKeys.forEach(function(depPath, index) {
+          if (index != 0) {
+              out += ",\n"; // separate to not have a trailing comma
+          }
+          out += "\"" + depPath + "\": " + depMap[canonicalSourcePath][depPath];
```

Comment:
doesn't this need to do more escaping? This code is hard to review and it's not tested

Reply:
Escaping: Fixed.

Testing: I have now tried this on these Debian packages:
node-base64-js node-debug node-deep-eql node-err-code node-timers-browserify theano
(theano embeds dagre, dagre-d3, graphlib and graphlib-dot; the others are single modules.)

All except theano were bit-identical before and after the change.  (This makes sense as they have <=1 dependencies: in dependency maps with 0-1 entries there is nothing to sort.)

theano had what looks like the intended change, i.e. sorted dependency maps, and its d3viz examples (the part that uses JavaScript; it's mostly a Python package) appeared to work in Firefox.